sweetalert2.d.ts

```diff
@@ -452,12 +464,30 @@ declare module 'sweetalert2' {
 
         /**
          * A custom CSS class for the modal.
+         * If a string value is provided, the classname will be applied to the popup.
```

Comment:
It works also with an array of strings. Do you want to add that?

Reply:
Done.